clients/ops/admin-ui/README.md

```diff
@@ -4,13 +4,13 @@ Admin UI for managing FidesOps privacy requests. A web application built in Next
 
 ## Running Locally
 
-1. Run `make server` in top-level `fidesops` directory, then run `make user` and follow prompts to create a user. Note that password requires 8 or more characters, upper and lowercase chars, a number, and a symbol.
+1. Run `docker-compose up` in top-level `fidesops` directory, then run `nox -s create_user` and follow prompts to create a user. Note that password requires 8 or more characters, upper and lowercase chars, a number, and a symbol.
```

Comment:
this command will spin up all of the related infra on its own, and I added the password requirements to the password prompt itself in a separate PR

```suggestion
1. Run `nox -s create_user` and follow prompts to create a user.
```